python/hsfs/core/training_dataset_engine.py

```diff
@@ -16,10 +16,23 @@
 
 import re
 from sqlalchemy import sql
-
-from hsfs import engine, training_dataset_feature, util
+import cloudpickle
+import mock
+
+try:
+    from pydoop import hdfs
+except ImportError:
+    hdfs = mock.Mock()
```

Comment:
Don't use Pydoop to get the file of the transformation functions, use the REST Api instead. We cannot use Pydoop on Databricks